index.js

```diff
@@ -84,13 +84,26 @@ class RPFormatter extends Formatter {
                 .map(attachment => attachment.body.replace(RP_ATTRIBUTE_PREFIX, ''));
             return [...new Set([...attachments, ...attrs])]
         }, []);
+
+        // convert collected attrributes to a key/value paired map
+        const mappedAttributes = attributes.reduce((listOfAttributes, item) => {
+            const splitItems = item.split(':');
+            const currentEntry = {
+                "key": splitItems[0],
+                "system": true,
+                "value": splitItems[1]
+            }
+            return [...new Set([...listOfAttributes, currentEntry])]
+        }, []);
+        console.log("String value: " + JSON.stringify(mappedAttributes));
```

Comment:
I need to remove this once the code works 100%. Just a self reminder, so I don't forget to remove it.